templates/php/src/Services/Service.php.twig

```diff
@@ -55,6 +55,39 @@ class {{ service.name | caseUcfirst }} extends Service
         }
 
 {% endfor %}
+{% if method.name == 'createFile' %}
+        $size = filesize($file);
+        //send single file if size is less than or equal to 5MB
+        if ($size <= Client::CHUNK_SIZE) {
+            $params['file'] = curl_file_create($file);
+                return $this->client->call(Client::METHOD_{{ method.method | caseUpper }}, $path, [
+{% for parameter in method.parameters.header %}
+                '{{ parameter.name }}' => ${{ parameter.name | caseCamel }},
+{% endfor %}
+{% for key, header in method.headers %}
+                '{{ key }}' => '{{ header }}',
+{% endfor %}
+                ], $params);
+        } else {
+            $handle = @fopen($file, "rb");
+            $fileId = null;
+            $mimeType = mime_content_type($file);
+            $counter = 0;
+            while (!feof($handle)) {
+                $params['file'] = new \CURLFile('data://' . $mimeType . ';base64,' . base64_encode(@fread($handle, Client::CHUNK_SIZE)), $mimeType, pathinfo($file, PATHINFO_FILENAME));
```

Comment:
Do we really need to encode it to base64? 🤔

Reply:
@TorstenDittmann I am unable to create `CURLFile` with a chunk, it directly creates from File, loading whole file, only way to load partial content is either to use this basc64 encoded string or create a chunk file for each chunk.

PHP 8.1 might solve this problem, there seems to have introduced new function `CURLFileFromString` or something